static/js/control.js

```diff
@@ -334,7 +334,7 @@ function updateViewByData(mo, cache) {
   updateCtrlByData(mo.data, mo.view);
   initInfoTable(mo.data, mo.view.spcols.len, mo.pick);
   initDataTable(mo.data.cols, mo.data.types);
-  fillDataTable(data);
+  fillDataTable(data, data.df.length);
```

Comment:
Yes, you can get it. But the issue is not that. In the fillDataTable function in control.js, it will default n to 100 if there is no second parameter. But if there is a df with less than 100 rows (let's say 50), there will be undefined values. When the variable i in the outer for loop (in the fillDataTable function)  becomes greater than the length of the df (so in this case 51), data.df[i] would be undefined, and line 1109 would return an error saying that "Uncaught TypeError: Cannot read properties of undefined (reading '0')". However, this only occurs when the length of the df is less than 100. So, if the web interface is supposed to enforce this condition then I will change the code to what it was originally.